Implemented individual ad page commment section using disqus

BrandonBalala opened the pull request:
#183

BrandonBalala committed:
implemented individual ad page commment section using disqus #183

kevcamel commented:
<img width="888" alt="screen shot 2017-11-02 at 12 31 13 am" src="https://user-images.githubusercontent.com/16027312/32309913-32a20126-bf65-11e7-96f5-3976fcb67572.png">

kevcamel commented on resources/views/layouts/app.blade.php:
```diff
@@ -129,5 +129,6 @@
 <script src="{{ asset('js/jquery/jquery.min.js') }}"></script>
 <script src="{{ asset('js/tether/tether.min.js') }}"></script>
 <script src="{{ asset('js/bootstrap/bootstrap.min.js') }}"></script>
+<script id="dsq-count-scr" src="//classifiedz-xyz.disqus.com/count.js" async></script>
```
what does the async do?

BrandonBalala replied:
I have no clue, but this line of code was given by disqus.

kevcamel commented on resources/views/viewAd.blade.php:
```diff
@@ -20,6 +20,29 @@
+        (d.head || d.body).appendChild(s);
+        })();
+        </script>
+        <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>                  
```
I had no idea this no script tag existed

neqqash replied:
neither did i, i guess we learn something new everyday

kevcamel reviewed (commented):
Good implementation using Disqus 👍 Lines 33-34 and 38-41 should be indented.

kevcamel commented:
@BrandonBalala once you indent those lines we could merge this PR

neqqash committed:
Update viewAd.blade.php

just indented the two functions (33-34) and (37-41). Did these indents with Java and made things much more visible.

neqqash commented:
Disqus FTW
